curriculum/challenges/english/02-javascript-algorithms-and-data-structures/object-oriented-programming/add-methods-after-inheritance.md

```diff
@@ -8,7 +8,7 @@ dashedName: add-methods-after-inheritance
 
 # --description--
 
-A constructor function that inherits its `prototype` object from a supertype constructor function can still have its own methods in addition to inherited methods.
+A `constructor` function that inherits its `prototype` object from a supertype `constructor` function can still have its own methods in addition to inherited methods.
```

Comment:
Not sure about wrapping "constructor" with backticks.  Would like to see what others think.

Reply:
I should not have wrapped this one - I did wrap the ones that specifically referred to a `constructor()` in the code, but this is not such a case.